extensions/cxf/deployment/src/main/java/io/quarkus/cxf/deployment/CxfConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.cxf.deployment;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+final class CxfConfig {
+
+    /**
+     * Set this to override the default path for CXF resources
+     */
+    @ConfigItem(defaultValue = "/")
+    String path;
+
+    /**
+     * Choose the path of each web services.
+     */
+    @ConfigItem(name = "endpoint")
+    Map<String, String> endpoints;
```

Comment:
I have a question here: we don't plan to add any per endpoint options? Because if we do, then it would be better to have a config object in the map with maybe an `implementor` config key so that we can add more later.

Not sure if it makes sense.

Reply:
Hmm good point, there are many annotation according to doc. https://cxf.apache.org/docs/annotations.html
Maybe in future, we can generate wsdl during deployment to avoid introspection but it is not link to endpoint. . Logging annotation, can be transfert to logger. There is interceptor which can be in config file but not link to a specific endpoint imo. I take look to configuration file and I found few features that can be linked to endpoint. Fail over load balancing can be linked to it or credential or wsdl location. Validation is annotation not linked to endpoint but to field. I see open api and swagger but seems not related to endpoint.

So to sumarize, enable logger, credential, wsdl location, fail over, load balancing can maybe be related to path.